tests/config_validator/test_validate.py

```diff
@@ -76,10 +76,22 @@ def test_validate_error_validation_target(mocker):
         config_validator.validate_deployment_files()
 
 
-def test_validate(mocker):
+def test_validate_applications(mocker):
     real_config_fixture_path = Path(
-        os.path.dirname(__file__), "..", "__fixtures__", "validator"
+        os.path.dirname(__file__), "..", "__fixtures__", "validator", "apps"
     ).resolve()
     mocker.patch.object(config_validator, "build_apps_path", return_value=real_config_fixture_path)
     mocker.patch.object(Configuration, "get_validation_target", return_value="APPLICATIONS")
     config_validator.validate_deployment_files()
+
+
+def test_validate_projects(mocker):
+    real_config_fixture_path = Path(
+        os.path.dirname(__file__), "..", "__fixtures__", "validator", "projects", "project.yaml"
+    ).resolve()
+    mocker.patch.object(config_validator, "build_apps_path", return_value=real_config_fixture_path)
+    mocker.patch.object(
+        config_validator, "build_project_conf_path", return_value=real_config_fixture_path
+    )
+    mocker.patch.object(Configuration, "get_validation_target", return_value="PROJECTS")
+    config_validator.validate_deployment_files()
```

Comment:
I find it a bit weird at first sight to not have an assert and I am afraid that the stacktrace might not be super helpful in case of an exception raised by `validate_deployment_files()`

What are your thoughts on doing the following?
```python
try:
    config_validator.validate_deployment_files()
except Exception as e:
    self.fail(f"validate_deployment_files() raised an unexpected Exception {err}")
```

Reply:
Yes I agree with you, I will add it to make the test clearer. FYI, we already thought of an improvement and it's to accumulate the errors for each file and not fail on the first invalid file. And if everything is fine we will assert on a more reasonable response.